activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/converter/ProcessModelConverterConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.cloud.services.organization.converter;
+
+import org.activiti.bpmn.converter.BpmnXMLConverter;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Configuration for process model validator
+ */
+@Configuration
+public class ProcessModelConverterConfiguration {
```

Comment:
I can add it in spring.factories, but I'm not sure about the benefits against how it is now.

Reply:
You're right, it will not make any difference now, it's more about having one thing less to migrate when we'll finally get rid of `@ComponentScan` annotations. In general we are using auto configuration as much as we can in new developments.

Reply:
I didn't know about the general approach. There are many @Configuration classes in modeling, maybe it makes sense to refactor all of them in bunch in a separate ticket.